### PR TITLE
Fix nested comments

### PR DESCRIPTION
**Pull request by sudden6**
**Description**

Nested comments produce  a warning when compiling, remove them. Also fixes trailing whitespace in this file.

### CODE REVIEW

File: src/grt/vhpi_user.h [hunk 1]
```diff
@@ -620,7 +620,7 @@ typedef enum {
 
 #ifdef VHPIEXTEND_INT_PROPERTIES
         VHPIEXTEND_INT_PROPERTIES
-        
+
```
Comment (Paebbels) [inline src/grt/vhpi_user.h:623]: Why is there a new empty line?

File: src/grt/vhpi_user.h [hunk 2]
```diff
@@ -652,7 +652,7 @@ typedef enum {
 
 #ifdef VHPIEXTEND_STR_PROPERTIES
         VHPIEXTEND_STR_PROPERTIES
- 
+
```
Comment (Paebbels) [inline src/grt/vhpi_user.h:655]: Why is there a new empty line?

**Review by Paebbels — CHANGES_REQUESTED** [review]

File: src/grt/vhpi_user.h (hunk 1 above)
Comment (tgingold) [reply src/grt/vhpi_user.h:623]: I don't think this is a new empty line, just trailing blank space removed.

Comment (Paebbels) [reply src/grt/vhpi_user.h:623]: Oh, now I get it.

File: src/grt/vhpi_user.h (hunk 2 above)
Comment (Paebbels) [reply src/grt/vhpi_user.h:655]: Oh, now I get it.

**Review by Paebbels — APPROVED** [review]

**Comment by Paebbels** [comment]
@tgingold LGTM.

But there is a failed job, because of black formatting rules.

**Comment by tgingold** [comment]
Thanks!

**Pull request merged**